### ISSUE TITLE
Autocomplete is failing badly for many cases

### ISSUE THREAD

**Issue by darrencroton** [argenos/nldates-obsidian#125]
It's currently `Thu 9th Feb`. Using Obsidian 1.1.9 on a Mac.

`@Friday` it returns `Fri 03 Feb` which is last Friday (expected: this Friday).
`@Next Week` returns `Tue 14 Feb` (expected: next Monday, or whatever the first day of the week is).
`@Next Monday` returns `Mon 06 Feb` (expected: next Monday).

... And so on. Many autocompletions work, but many don't. Hopefully these can be fixed. Thanks for your great work on this!

**Comment by darrencroton** [comment]
I found a partial solution (indicating a bug). 

I had first day of the week set to Monday in the settings. If I change this back to "Locale default (sunday)", `@Next Monday` now works correctly. But unfortunately `@Next Week` returns Sunday (obviously) which isn't what I want (my first day of the week is Monday). But I can live with that.

So there seems to be a bug parsing this setting with the autocomplete dates.

**Comment by AkselNLadegaard** [comment]
I'm also experiencing the same issue, the same fix 'works' for me, but it'd be vastly more useful to be able to use ones preferred day (Monday) to start the week.

**Comment by InvaderJ** [comment]
Yeah I can echo this being totally broken. I've resorted to just using "next Friday" when I actually mean "Friday".  -__-
